portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +26,70 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function expandDesign(clickedId){
+//whenever you click an image on the description page, this pulls up more images in the category alongside descriptions
+const designPage = document.getElementById('design_container');
+designPage.innerHTML = "";
+console.log(clickedId); //REMOVE WHEN DONE
+
+//descriptions for every single page that can be clicked on
+const descriptions = 
```

Comment:
why are we putting the description in a list? My guess is so we can save multiple descriptions for multiple projects here. If that is the case, we might be better off doing specifically named variables. fake_band_description is a much clearer name than descriptions[0].

also, you can wrap the text in `` to do multiline strings in javascript.